test/e2e/storage/csi_volumes.go

```diff
@@ -158,6 +160,17 @@ var _ = Describe("PMEM Volumes", func() {
 			// create and test volumes in parallel until
 			// we've tested m volumes.
 
+			// Because this test creates a lot of pods, it is useful to
+			// log their progress.
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+			to := podlogs.LogOutput{
+				StatusWriter: GinkgoWriter,
+				LogWriter:    GinkgoWriter,
+			}
+			podlogs.CopyAllLogs(ctx, f.ClientSet, f.Namespace.Name, to)
+			podlogs.WatchPods(ctx, f.ClientSet, f.Namespace.Name, GinkgoWriter)
```

Comment:
As the test itself is pending(Pit) test, does this change have any effect now.

Reply:
No, but it will once someone runs the test manually. IMHO it's useful to have.